mqtt/mqtt-broker/Cargo.toml

```
@@ -10,11 +10,15 @@ edition = "2018"
 async-trait = "0.1"
 bincode = "1.2"
 chrono = "0.4"
+config = "0.10"
```

```suggestion
config = { version = "0.10", features = ["json"] }
```
to disable other file formats